Applications/TransferLearning/import_pb_to_tensorboard.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at #
+# http://www.apache.org/licenses/LICENSE-2.0 #
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ================================
+
+# @file	import_pb_to_tensorboard.py
+# @date	09 Sept 2020
+# @brief	This is generate data for tensorboard
+#
+
+"""Imports a protobuf model as a graph in Tensorboard."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import argparse
+import sys
+from tensorflow.core.framework import graph_pb2
+from tensorflow.python.client import session
+from tensorflow.python.framework import importer
+from tensorflow.python.framework import ops
+from tensorflow.python.platform import app
+from tensorflow.python.platform import gfile
+from tensorflow.python.summary import summary
+
+# Try importing TensorRT ops if available
+# TODO(aaroey): ideally we should import everything from contrib, but currently
```

Comment:
if this is imported from other location, let's say that explicitly
and let's clean up these unnecessary comments

Reply:
ditto.